core/tests/protractor_utils/TopicAndStoryViewerPage.js

```diff
@@ -40,8 +43,12 @@ var TopicAndStoryViewerPage = function() {
     await waitFor.pageToFullyLoad();
   };
 
-  this.expectLockedChaptersCountToBe = async function(count) {
-    expect(await lockedChapterList.count()).toEqual(count);
+  this.expectCompletedLessonCountToBe = async function(count) {
+    expect(await lessonCompletedIcons.count()).toEqual(count);
+  };
+
+  this.expectUncompletedLessonCountToBe = async function(count) {
+    expect(await lessonUncompletedIcons.count()).toEqual(count);
```

Comment:
Can you add waitFor calls here for when `count != 0`?

Reply:
Yup, I've added a `waitFor.visibilityOf` for the parent container now. Thanks!